src/java/org/apache/cassandra/db/streaming/ComponentManifest.java

```diff
@@ -106,6 +106,14 @@ public int hashCode()
         return components.hashCode();
     }
 
+    @Override
+    public String toString()
```

Comment:
we log the manifest, but since there was no tostring it just is the pointer... so fixed so logs mean something